bin/print_build_env.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+from string import Template
+import subprocess
+import os
+import re
+import contextlib
+
+##
+# Prints some details about the current build environment.
+#
+# Example Usage:
+#   ./print_build_env.py
+#
+
+BIN_DIR = os.path.dirname(__file__)
+webrtc_src_dir = os.path.join(BIN_DIR, '../webrtc/src')
+
+@contextlib.contextmanager
+def pushd(new_dir):
+    previous_dir = os.getcwd()
+    os.chdir(new_dir)
+    yield
+    os.chdir(previous_dir)
+
+def determine_git_branch(directory):
+    with pushd(directory):
+        git_branch_output = subprocess.check_output(["git", "branch"])
+        git_branch = [line.replace("* ", "") for line in git_branch_output.split("\n") if re.search("^\*", line)][0]
+        return git_branch
+
+def get_build_details():
+    template = Template("""## WebRTC Build Details
+
+To track down potential future issues, we log some of our build environment details.
+
+git branch:
+$git_branch
```

Comment:
git commit too?

Reply:
Done! 

I'm now capturing two sha's, one for the upstream webrtc repo, and one for the build tools (this repo)